src/pages/index.tsx

```diff
@@ -1,87 +1,76 @@
+        <Flex bgColor="purple" color="white" direction="column"  p={["24px", "64px"]}>
+          <Stack spacing="44px">
+            <Heading fontSize={["16px", "24px"]} fontWeight="normal">
               Internet in NYC is essential, but expensive, slow and segregated.
             </Heading>
-            <Heading fontFamily="Inconsolata" size="md" fontWeight="500">
-              The Internet For All coalition is fighting for a publicly owned, fully-unionized municipal broadband provider to make sure every New Yorker can access education, work, and other essential services online
+            <Heading fontSize={["16px", "24px"]} fontWeight="normal">
+              The Internet For All coalition is fighting for a publicly owned, fully-unionized municipal broadband
+              provider to make sure every New Yorker can access education, work, and other essential services online.
             </Heading>
           </Stack>
-          <Flex justifyContent="flex-end" mt="3rem">
-            <PrimaryButton
-              href="/take-action"
-              title="Get Involved"
-              background="orange"
-              backgroundHover="orangeLight"
-              color="offBlack"
-            />
+          <Flex justify="flex-end" mt="44px">
+            <Link as={GatsbyLink} to="/take-action">
+              <PrimaryButton background="orange" backgroundHover='orangeLight'>
+                Get Involved
+              </PrimaryButton>
+            </Link>
           </Flex>
         </Flex>
-      </Center>
-      <Box bgColor="offWhite" p={['24px', '64px']} py={['36px', '64px']} w="100%" textAlign="center">
-        <Container maxWidth="1185px">
-          <Heading size="md" fontWeight="500">Internet is essential</Heading>
-          <SimpleGrid columns={[1, null, 3]} spacing={[5, 10]} mt={['24px', '64px']}>
-            <Box>
-              <Heading color="purple" size="2xl" p={2}>79%</Heading>
-              <p>of Americans used internet for their last job search</p>
-            </Box>
-            <Box>
-              <Heading color="orange" size="2xl" p={2}>50%</Heading>
-              <p>of students say they need internet access to do their homework</p>
-            </Box>
-            <Box>
-              <Heading color="blue" size="2xl" p={2}>86%</Heading>
-              <p>of Americans get news from digital devices</p>
-            </Box>
-          </SimpleGrid>
-        </Container>
+      <Box bgColor="offWhite" w="100%"  textAlign="center" p={["24px", "64px"]}>
+        <Heading fontSize={24} fontWeight={500}>Internet is essential</Heading>
+        <Wrap spacing={["72px", "160px"]} mt={['24px', '64px']} justify="center">
+          <VStack spacing={"32px"} w={256}>
+            <Heading color="purple" fontSize={64} fontWeight={900}>79%</Heading>
+            <Text>of Americans used internet for their last job search</Text>
+          </VStack>
+          <VStack spacing={"32px"} w={256}>
```

Comment:
Having a fixed width for these makes them look more consistent imo since the different text takes up different amounts of space